spec/queueSpec.js

```diff
@@ -0,0 +1,69 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+describe('Queue', () => {
+  'use strict'
```

Comment:
look into beforeEach so you don't have to create a new object with every context 
`  let myQueue
  beforeEach(() => {
    myQueue = new Queue()
  })`